code/game/objects/items/clown_items.dm

```diff
@@ -207,3 +207,11 @@
 	desc = "Just looking at this makes you want to giggle."
 	icon_state = "laughter"
 	list_reagents = list("laughter" = 50)
+
+//Honk-MAG, great fuckin idea @Orangebottle

```

Comment:
No player references in code.

Reply:
i once tried to sign when i made CQC and turns out no you cant do that, just use git history